### PR TITLE
Test lossy (non-pfc) queue for cisco-8000 platform

### PR DESCRIPTION
**Pull request by jsanghra**
### Description of PR
Adding new test for lossy (non-pfc) queue for cisco-8000 platform .

#### What is the motivation for this PR?
Verify traffic on lossy queues .

#### How did you verify/test it?
Verified this testcase on local cisco-8000 setup

This PR has dependency on https://github.com/sonic-net/sonic-mgmt/pull/5982
Once the above PR is merged , then this PR should be merged

### CODE REVIEW

File: tests/qos/test_qos_sai.py [hunk 1]
```diff
@@ -723,6 +724,71 @@ def testQosSaiLossyQueue(
...
+        """
+        if dutTestParams["basicParams"]["sonic_asic_type"] != "cisco-8000":
+            pytest.skip("Lossy Queue Voq test is not supported")
+        portSpeedCableLength = dutQosConfig["portSpeedCableLength"]
+        qosConfig = dutQosConfig["param"][portSpeedCableLength]
+        testPortIps = dutConfig["testPortIps"]
+        
+        if "lossy_queue_voq_2" in LossyVoq :
+            original_voq_markings = get_markings_dut(duthost)
+            setup_markings_dut(duthost, localhost, voq_allocation_mode = "default")
```
Comment (kevinskwang) [inline tests/qos/test_qos_sai.py:757]: So any case to cover the voq_allocation_mode is src_port_flow_hash?

File: tests/saitests/sai_qos_tests.py [hunk 2]
```diff
@@ -2134,6 +2135,133 @@ def runTest(self):
...
+            cell_size = int(self.test_params['cell_size'])
+            if packet_length != 64:
+                cell_occupancy = (packet_length + cell_size - 1) / cell_size
+                pkts_num_trig_egr_drp /= cell_occupancy
+        else:
+            packet_length = 64
+
+        pkt_dst_mac = router_mac if router_mac != '' else dst_port_mac
+        # crafting 2 udp packets with different udp_dport in order for traffic to go through different flows
+        pkt = simple_udp_packet(pktlen=packet_length,
```
Comment (kevinskwang) [inline tests/saitests/sai_qos_tests.py:2176]: As the voq mode is default, could you indicate the reason why we need go through the different flows using two different udp pkt?

File: tests/qos/test_qos_sai.py (hunk 1 above)
Comment (jsanghra) [reply tests/qos/test_qos_sai.py:757]: Yes , so voq_allocation_mode will be always src_port_flow_hash .
For the lossy_queue_voq_2 , we change the voq_allocation_mode to default and then at the end of testcase revert it back to src_port_flow_hash

**Comment by jsanghra** [comment]
Fixed the unused variable lgtm-com alert errors

File: tests/saitests/sai_qos_tests.py (hunk 2 above)
Comment (jsanghra) [reply tests/saitests/sai_qos_tests.py:2175]: We have kept 2 flows and reduced the number of packets to half in default voq case to keep the testcase uniform . Both flows will go on same voq .

**Comment by sachinv-msft** [comment]
@kevinskwang @jsanghra where are we with this one?

**Comment by jsanghra** [comment]
Hi Sachin ,
There were some Azure pipeline run cases failing which we believe are not related to our testcase changes so Kevin had asked Xin Wang to check that . There was an email thread discussion where Raghav had sent out the issue we were hitting .

**Comment by jsanghra** [comment]
Raised a New PR with rebased code :
https://github.com/sonic-net/sonic-mgmt/pull/6562

**Pull request closed without merging**